src/Features/LanguageServer/Protocol/WellKnownLspServerKinds.cs

```diff
@@ -96,6 +104,7 @@ public static string GetContractName(this WellKnownLspServerKinds server)
     {
         return server switch
         {
+            WellKnownLspServerKinds.RazorCohostServer => "RazorLanguageServer",
```

Comment:
Can we define the contract in the same protocol constants type, and also reference the protocol constants in the razor external access`Constants` class?